libs/solid/skeleton/src/lib/components/feedback/feedback.component.ts

```diff
@@ -43,18 +55,43 @@ export class FeedbackComponent implements OnInit, OnDestroy {
   }
 
   public ngOnDestroy(): void {
+    /** 
     sessionStorage.setItem(
       FeedbackComponent.STORAGE_KEY,
       JSON.stringify(this.Form.value)
     );
+    */
+    // why do we need this step ?
```

Comment:
Der Inhalt des Feedback-Formulars wird in den Session-Storage geschrieben. 
Wenn das Formular abgebrochen wird (z.B. durch einen Klick auf den Hintergrund) und dann wieder aufrufe, sind meine Daten nicht verloren.

Ich denke, das ist eine Convenience-Sache, wenn ich z.B. ein langes Feedback schreibe und dann versehentlich das Fenster schließe, sollte nicht alles verloren sein. Ich würde es daher drin lassen - es sollte unsere Änderungen nicht stören.